Formula/libarchive.rb

```diff
@@ -15,15 +15,48 @@ class Libarchive < Formula
   end
 
   keg_only :provided_by_osx
+  # Note there are now `--enable-bsd[tar|cpio|cat]=static` options which
+  # allow the executables to be easily separated from the library install. In
+  # addition, `bsdcat` is a new tool, similar to `[z|bz|xz]cat` but for *all*
+  # libarchive supported formats. This tool will be unavailable by default,
+  # because this formula is keg-only. Moreover, there are literally hundreds
+  # of improvements (e.g., support for HFS compression, support for multiple
+  # filters) and half a dozen or so new formats (e.g., rar, lz4, cab, 7z)
+  # supported by this [bsd]tar over Apple's ancient libarchive 2.8.3 version.
+  #
+  # Thus it may be prudent to create a separate "bsdtar" formula based on the
+  # same package (there is precedent for this, i.e., libmagic.rb and
+  # file-formula.rb, though I'd argue for both to be in homebrew-core in
+  # this case), allowing these tools to shadow their inferior system
+  # counterparts, without interfering with linking to the system libarchive.
+  # There is precedence for shadowing some tools: for example, the `awk`
+  # symlink installed with the gawk formula is allowed to shadow
+  # `/usr/bin/awk`. And there's ruby, python, bash, etc.
 
-  depends_on "xz" => :optional
+  depends_on "xz"   => :recommended
+  depends_on "lz4"  => :recommended
+  depends_on "lzop" => :recommended
```

Comment:
I don't think it's worth making these all recommended; makes this formula quite a bit heavier.